openshift/tests-extension/pkg/helpers/commands.go

```diff
@@ -0,0 +1,33 @@
+package helpers
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+)
+
+// findK8sClient returns "oc" if available, otherwise "kubectl".
```

Comment:
The name of this function is a little too close to the client used for the other tests:
https://github.com/openshift/operator-framework-operator-controller/blob/2f8f54b35eb3cc626f9693bf23c0b475bdbdca54/openshift/tests-extension/pkg/env/cluster.go#L37